### PR TITLE
[bug 1032455] Fix post-merge issues

### PR DESCRIPTION
**Pull request by hoosteeno**

### CODE REVIEW

File: kuma/wiki/templates/wiki/document.html [hunk 1]
```diff
@@ -416,7 +416,10 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
       {# This can be removed once the social network button multivariate test is completed. #}
       mdn.optimizely.push(['activate', 2796520257]);
   </script>
-  {% if waffle.flag('share_links') or waffle.flag('helpfulness') %}
-    {{ js('document') }}
+  {% if waffle.flag('share_links') %}
+    {{ js('social') }}
+  {% endif %}
```
Comment (jezdez) [inline kuma/wiki/templates/wiki/document.html]: In case those flags are exclusive there is also `{% elif %}`

Comment (hoosteeno) [inline kuma/wiki/templates/wiki/document.html]: Both/either/neither are all valid. That's why the original code didn't work -- it coupled them.

Comment (jezdez) [inline kuma/wiki/templates/wiki/document.html]: Understood :+1:

**Comment by stephaniehobson** [comment]
R+

**Pull request merged**